### DISCUSSION TITLE
Call for topics from the MNF TSC

### DISCUSSION THREAD

**Discussion by BalestraPatrick** [MobileNativeFoundation/discussions#148] [Proposals]
Hello everyone!

As part of our MNF monthly TSC meetings, we are trying to come up with a list of topics that companies and members would be interested in discussing and learning. We have seen some great threads lately in this forum, and we would like to continue to dive deeper in these discussions in other ways (under which form this will be is still TBD).

Coming up with a list of topics and interest degree will help us set the priority of what problems to tackle first. Some of the topics might for example include:

- Declarative UI frameworks usage (SwiftUI, Jetpack Compose, etc.)
- Repository layout
- Build systems
- Testing practices
- etc.

Please feel free to comment with your interest in the above topics and other new topics that you'd like to add. We would also be glad if you can elaborate on the reason of your interest in a topic to better understand the problems currently at the heart of the community.

**Comment by bootstraponline** [comment]
I think an annual survey would be great. The CNCF survey is [one example](https://www.cncf.io/wp-content/uploads/2020/11/CNCF_Survey_Report_2020.pdf). Understanding what tech is working well across different companies will save people time.

**Comment by piannaf** [comment]
All of the above :) 

But especially 

1. The intersection of multiplatform solutions and build systems, testing, coordinating feature development, maintenance, etc https://github.com/MobileNativeFoundation/discussions/discussions/43
2. The intersection of Declarative UI and SDUI https://github.com/MobileNativeFoundation/discussions/discussions/47

(1) for the age-old questions of accelerating development/fixes, reducing risk, etc. But also because going this route impacts how teams work at scale that could cause more problems than it solves. (I'm partial to Kotlin Multiplatform because it is the most Native of the options)

(2) because it opens opportunities to better leverage design systems (for all the benefits those have) and deploy changes without an app store/play store download, whether rendering Native UI or otherwise. (I'm partial to Native UI)

**Comment by shams-ahmed** [comment]
I would like to see a discussion on testing practices and build systems, mainly at scale. i.e best practices, limitations, workarounds etc..

**Comment by ferologics** [comment]
Using Kotlin Multiplatform for building platforms with shared business logic would be interesting. And perhaps how that could extend to BE. Mobile seems to be standardizing now.
